CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ and this project adheres to
 - Fix `printf` not allowing format specifiers to be directly followed by
   alphabetic characters
   - [#1414](https://github.com/iovisor/bpftrace/pull/1414)
+- Add feature specific runtime testcase support
```

Comment:
I don't think we should add internal things like these test changes to the changelog as they'ree not relevant to end users.

Reply:
OK, I'll remove it.